templates/pre_reqs.sh

```diff
@@ -19,6 +19,7 @@ function ubuntu_pre_reqs {
     DOCKER_VERSION=`sudo apt-cache madison docker-ce | grep '19.03.13' | awk '{print $3}'`
     sudo apt-get install docker-ce=$DOCKER_VERSION -qy
     sudo usermod -aG docker $USER
+    sudo apt-get install -qy lvm2
```

Comment:
This can probably be removed since we are not changing the disks on the control plane nodes.